client/src/main/java/model/FleetMapper.java

```diff
@@ -0,0 +1,20 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FleetMapper {
+
+    public FleetModel mapToFleetModel(Fleet fleetToMap) {
+        List<ShipModel> ships = new ArrayList<>();
+        ShipMapper shipMapper = new ShipMapper();
+
+        List<Ship> shipsToMap = fleetToMap.getShips();
+
+        for (Ship ship:shipsToMap) {
```

Comment:
Consider replacing with stream :) But only consider if you don't want don't change it